<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageConfigDlg</class>
 <widget class="QWidget" name="PageConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>668</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>PageConfigDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>页面设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>30</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="pageNameHorizontalLayout">
        <item>
         <widget class="QLabel" name="pageNameLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>页面名称</string>
          </property>
          <property name="buddy">
           <cstring>pageNameLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pageNameLineEdit">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="meterGroupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>仪表显示方式</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="rowLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>行数</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
           <property name="buddy">
            <cstring>rowSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="rowSpinBox">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="columnLabel">
           <property name="text">
            <string>列数</string>
           </property>
           <property name="buddy">
            <cstring>columnSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="columnSpinBox">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="lampGroupBox">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="title">
         <string>指示灯显示方式</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lampLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>指示灯数目</string>
           </property>
           <property name="buddy">
            <cstring>lampNumSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="lampNumSpinBox">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="maximum">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="posAdjustPushButton">
        <property name="text">
         <string>调整仪表位置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>366</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>meterGroupBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lampGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>lampNumSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lampGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>lampNumSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
